### PR TITLE
Removing duplicate config entries and typos

### PR DESCRIPTION
**Pull request by bchavez**
Some typos and duplicate config entries in the sample freelan.cfg

### CODE REVIEW

File: apps/freelan/config/freelan.cfg [hunk 1]
```diff
@@ -779,11 +779,6 @@ ipv6_address_prefix_length=2aa1::1/8
 # Default: 2000
 #passphrase_iterations_count=2000
 
-# The salt to use when deriving the PSK from the passphrase.
-#
-# Default: freelan
-#passphrase_salt=freelan
```
Comment (ereOn) [inline apps/freelan/config/freelan.cfg]: You removed the whole section for the `passphrase_salt` was that intended ?

This is not the same as `passphrase`.

**Comment by ereOn** [comment]
Thanks for this !

File: apps/freelan/config/freelan.cfg (hunk 1 above)
Comment (bchavez) [inline apps/freelan/config/freelan.cfg]: `passphrase_salt` is defined twice: Line 771 and Line 785 in the original file. It removes the duplicate `passphrase_salt`

Comment (bchavez) [inline apps/freelan/config/freelan.cfg]: Specifically here :+1: 

https://github.com/freelan-developers/freelan-all/blob/master/apps/freelan/config/freelan.cfg#L771

https://github.com/freelan-developers/freelan-all/blob/master/apps/freelan/config/freelan.cfg#L785

**Pull request merged**